webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -392,7 +392,11 @@ object RudderConfig extends Loggable {
     config.getString("rudder.git.gc")
   } catch {
     // missing key, perhaps due to migration, use default
-    case ex: Exception => "0 42 3 * * ?"
+    case ex: Exception => {
+      val default = "0 42 3 * * ?"
+      logger.info(s"`rudder.git.gc` property is missing, using default schedule: ${default}")
```

Comment:
let user be able to discover that he can change git gc schedule with that property